src/tests/system/tests/test_logging.py

```diff
@@ -45,38 +50,92 @@ def test_logging__default_settings_logs_debug_level(client: Client):
 @pytest.mark.integration
 @pytest.mark.importance("low")
 @pytest.mark.topology(KnownTopology.Client)
-def test_logging__default_settings_does_not_log_user_logins(client: Client):
+def test_logging_debug_level_is_written_to_logs(client: Client):
+    """
+    :title: Domain log level from configuration is honored
+    :setup:
+        1. Configure SSSD for local system authentication
+        2. Clear logs and start SSSD with debug level 9
+    :steps:
+        1. Check log files
+    :expectedresults:
+        1. Logs messages contain configured debug level 0x4000
+    :customerscenario: False
+    """
+    client.sssd.common.local()
+    client.sssd.domain["debug_level"] = "9"
+    client.sssd.clear(logs=True)
+    client.sssd.start(debug_level=None)
+    log_str = client.fs.read(client.sssd.logs.domain())
+
+    assert "level = 0x2f7f0" in log_str, "Logs should contain debug level = 0x2f7f0!"
+
+
+@pytest.mark.integration
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.Client)
+def test_logging_change_debug_level_during_runtime(client: Client):
+    """
+    :title: Domain log level set in runtime is honored
+    :setup:
+        1. Configure SSSD for local system authentication
+        2. Clear logs and start SSSD with defaut level
+    :steps:
+        1. Check log file
+        2. Change domain debug level in runtime using sssctl debug-level
+        3. Check log file again
+    :expectedresults:
+        1. Logs messages contain default debug level 0x0070
+        2. Debug level change command succeeds
+        3. Logs messages contain default debug level 0x2f7f0
+    :customerscenario: False
+    """
+    client.sssd.common.local()
+    client.sssd.clear(logs=True)
+    client.sssd.start(debug_level=None)
+
+    log_str = client.fs.read(client.sssd.logs.domain())
+    sssctl_set = client.host.conn.run("sssctl debug-level --domain local 9")
+    sssctl_get = client.host.conn.run("sssctl debug-level --domain local")
+
+    assert "level = 0x0070" in log_str, "Logs should contain debug level = 0x0070!"
+    assert sssctl_set.rc == 0, "'sssctl debug-level' command failed!"
+    assert sssctl_get.rc == 0, "'sssctl debug-level' command failed!"
+    assert " 0x2f7f0" in sssctl_get.stdout, "Sssctl debug-level output should contain 0x2f7f0"
```

Comment:
space? 0x2f7f0, does just the string return more than one result?

'Sssctl' is strange since it's a command, 'sssctl'

and an exclamation at the end

Reply:
done